archaius2-archaius1-bridge/src/main/java/com/netflix/archaius/bridge/StaticAbstractConfiguration.java

```diff
@@ -35,21 +59,37 @@ public static void reset() {
 
     @Override
     public boolean isEmpty() {
+        if (delegate == null) {
+            System.err.println("[isEmpty()] StaticAbstractConfiguration not initialized yet.");
```

Comment:
Due to the excessive use of statics in various libraries (blitz4j) there are some unavoidable situations where the configuration system hasn't been set up yet and it would be impossible to move forward if this code threw an exception.  For now I'm opting to output an error and slowly fix libraries that have these issues.  In most cases the libraries have very reasonable defaults or get reconfigured later on.